Quaver.Shared/QuaverGame.cs

```diff
@@ -185,6 +185,31 @@ public string Version
         /// </summary>
         private bool WindowActiveInPreviousFrame { get; set; }
 
+        private event EventHandler WindowFocusChanged;
```

Comment:
Instead of this, I would create a special type of EventArgs for this which contains the new focus value. It would allow subscribers of the event to have access to the new value from anywhere.

```cs
private event EventHandler<FocusChangeEventArgs> WindowFocusChanged;
```



```cs
public class FocusChangeEventArgs : EventArgs
{
    public bool IsFocused { get; }

    public FocusChangeEventArgs(bool isFocused) => IsFocused = isFocused;
}
```